src/components/Modeler.vue

```diff
@@ -688,8 +688,8 @@ export default {
       const diagram = this.nodeRegistry[control.type].diagram(this.moddle);
 
       // Handle transform
-      const paperOrigin = this.paper.localToPagePoint(0, 0);
-      const scale = this.paper.scale();
+      const paperOrigin = this.paperManager.paper.localToPagePoint(0, 0);
+      const scale = this.paperManager.paper.scale();
```

Comment:
You can use the class getter:
```suggestion
      const scale = this.paperManager.scale;
```

Reply:
I missed this call, I'll change it now. Thanks.